src/Support/FileRemover/FileRemover.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\MediaLibrary\Support\FileRemover;
+
+use Illuminate\Contracts\Filesystem\Factory;
+use Spatie\MediaLibrary\MediaCollections\Filesystem;
+use Spatie\MediaLibrary\MediaCollections\Models\Media;
+
+interface FileRemover
+{
+    /*
+     * Create a new file remover instance, using the media library file system and illuminate filesystem.
+     */
+    public function __construct(Filesystem $mediaFileSystem, Factory $filesystem);
+
+    /*
+     * Remove all files relating to the media model.
+     */
+    public function removeAllFiles(Media $media): void;
```

Comment:
Rename this to `removeOriginalFiles`

Reply:
This name matches the existing method name inside the Filesystem class 
```php
public function removeAllFiles(Media $media): void
```